src/ts/src/common/utils/tableUtils.tsx

```diff
@@ -1,12 +1,10 @@
 /* eslint-disable react/display-name */
 
-import React from "react";
-
 const UNDEFINED_TEXT = "---";
 
 function createTableCellRenderer(
   customRenderers: Record<string | number, CellRenderer>,
-  customStyle?: Stylesheet
```

Comment:
The merge conflict here was harder to sort out, but my guess is that the API of `createTableCellRenderer()` should now accept `(customRenderers, defaultRenderer)` instead of `(customRenderers, customStyle)`?